app/components/filters/select-multiple-filter.ts

```diff
@@ -1,29 +1,58 @@
 import { action, get } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
-import {
-  deserializeArray,
-  serializeArray,
-} from 'frontend-burgernabije-besluitendatabank/utils/query-params';
+import { deserializeArray } from 'frontend-burgernabije-besluitendatabank/utils/query-params';
 import FilterComponent, { type FilterArgs } from './filter';
 
 type Option = Record<string, string>;
 
+type GroupedOptions = {
+  groupName: string;
+  options: Option[];
+};
+
 interface Signature {
   Args: {
+    route?: string;
     options: Promise<Option[]>;
   } & FilterArgs;
 }
 
 export default class SelectMultipleFilterComponent extends FilterComponent<Signature> {
   @tracked selected?: Option[];
 
-  /** Action to parse the queryParameter value(s) & select them in the select input */
   @action
   async inserted() {
-    // The queryParam is an array of searchField values, joined by seperator
-    // On the page load, we can use this to load in values
-    const queryParam = this.getQueryParam(this.args.queryParam);
-    if (queryParam) {
+    if (this.args.queryParam?.includes(',')) {
+      const queryParams = this.getQueryParam(this.args.queryParam) as string;
+      const needles = queryParams.split(/[+,]/);
```

Comment:
**Question**: When is `queryParams` contains ","?

Reply:
I've made it so that you can add multiple query params delimited by a comma (,) in the multiple select

Reply:
I'm not sure to understand, isn't it what the `+` is use for? Do you have an example?

Reply:
Nevermind! I am always a bit confused when talking about query params, I thought this.args.queryParam was the variable instead of the param name. I've changed the split to deserialize now!